tests/data/dfc2022/data.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import os
+import shutil
+import random
+import hashlib
+
+import rasterio
+import numpy as np
+
+from torchgeo.datasets import DFC2022
+
+ROOT = "/path/to/dataset"
+NUM_SAMPLES = 2
+SIZE = 32
+
+np.random.seed(0)
+random.seed(0)
+
+
+def create_file(path: str, output_path: str) -> None:
+    src = rasterio.open(path)
```

Comment:
This assumes that the data already exists. We don't want this, as it disallows us from generating data on-the-fly in the future. You should hardcode the metadata in `data.py` like I do in other `data.py` scripts.

Reply:
Fixed